wls-vorfaelleundvorkommnisse-service/src/main/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/service/EreignisModel.java

```diff
@@ -0,0 +1,12 @@
+package de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.service;
+
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.domain.ereignis.Ereignisart;
+import jakarta.validation.constraints.NotNull;
+import lombok.Builder;
+
+@Builder
+public record EreignisModel(@NotNull String wahlbezirkID,
+                            String beschreibung,
+                            java.time.LocalDateTime uhrzeit,
```

Comment:
dont use fully qualified name

Reply:
there are more occasions where the full qualified name is called, pleas search for them by yourself

Reply:
see commit [f58e2b5](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/f58e2b5b367a44e8f46391ea078a62e4a1bcf65a)